solc/CommandLineInterface.cpp

```diff
@@ -809,7 +809,7 @@ bool CommandLineInterface::processInput()
 		return link();
 	}
 
-	m_compiler.reset(new CompilerStack(fileReader));
+	m_compiler.reset(new CompilerStack(fileReader, inputLanguage));
```

Comment:
inputLanguage is only relevant for the `assemble` call above which does not use `CompilerStack`